polkadot/node/core/pvf/src/host.rs

```diff
@@ -130,12 +130,33 @@ impl ValidationHost {
 			.await
 			.map_err(|_| "the inner loop hung up".to_string())
 	}
+
+	/// TEST-ONLY: deletes all on-disk artifacts and resets the artifacts table.
+	#[cfg(feature = "test-utils")]
+	pub async fn prune_all_artifacts(
+		&mut self,
+		result_tx: oneshot::Sender<Result<usize, std::io::Error>>,
+	) -> Result<(), String> {
+		self.to_host_tx
+			.send(ToHost::PruneAllArtifacts { result_tx })
+			.await
+			.map_err(|_| "the inner loop hung up".to_string())
+	}
 }
 
 enum ToHost {
-	PrecheckPvf { pvf: PvfPrepData, result_tx: PrepareResultSender },
+	PrecheckPvf {
+		pvf: PvfPrepData,
+		result_tx: PrepareResultSender,
+	},
 	ExecutePvf(ExecutePvfInputs),
-	HeadsUp { active_pvfs: Vec<PvfPrepData> },
+	HeadsUp {
+		active_pvfs: Vec<PvfPrepData>,
+	},
+	#[cfg(feature = "test-utils")]
+	PruneAllArtifacts {
```

Comment:
It shouldn't end up in the production binary since it's gated behind `test-utils`. Is that fine? As per my comment above, there's not really great alternatives.